src/java.xml/share/conf/jaxp-strict.properties

```diff
@@ -0,0 +1,129 @@
+################################################################################
+#           JAXP Strict Configuration File
+#
+# This property file, jaxp-strict.properties, provides settings that will be
+# equivalent to what will be the default JAXP settings in a future release to
+# make the use of JAXP more secure by default. This file allows deployments to
+# test the more secure/strict behavior, identify issues such as a processor
+# unknowingly makes outbound network connections to fetch DTD, or processes XML
+# that relies on extension functions.
```

Comment:
There isn't a JEP to propose that XML processing be secure by default on the technical roadmap right now so I think this paragraph will need to be tweaked to avoid making any assumptions. I think just say that the file provides the settings for more secure XML processing and drop the text about testing (and "and create your own configuration file for the experiment" from the paragraph below).